src/layer/marker/Marker.js

```diff
@@ -217,17 +221,14 @@ export var Marker = Layer.extend({
 			}
 
 			if (icon.tagName === 'IMG') {
-				icon.alt = options.alt || '';
+				icon.alt = options.alt || 'Location';

```

Comment:
> Instead of hardcoding some arbitrary string you better change `options.alt` defaults.

_Originally posted by @johnd0e in https://github.com/Leaflet/Leaflet/pull/6248#r412840042_

Reply:
> "location" seems opinionated, and may not make sense in a non-geographic map. How about "Marker"?

_Originally posted by @Malvoz in https://github.com/Leaflet/Leaflet/pull/6248#r563012140_

Reply:
The default - as @johnd0e suggested - was change to `Marker` in [#7895](https://github.com/Leaflet/Leaflet/pull/7895/files#diff-9f4983a1c995bb3bbfe108613622e7270372359622822ba3c3cbbf3359bcc3f4R45).